src/lib/modules/chat.ts

```diff
@@ -159,3 +167,15 @@ module.exports.player = function (player, serv) {
     player._client.write('chat', { message: JSON.stringify(message), position: 2, sender: '0' })
   }
 }
+declare global {
+  interface Server {
+    "broadcast": (message: any, { whitelist, blacklist, system }?: { whitelist?: any; blacklist?: any[] | undefined; system?: boolean | undefined }) => void
+    "color": { black: string; dark_blue: string; dark_green: string; dark_cyan: string; dark_red: string; purple: string; dark_purple: string; gold: string; gray: string; grey: string; dark_gray: string; dark_grey: string; blue: string; green: string; aqua: string; cyan: string; red: string; pink: string; light_purple: string; yellow: string; white: string; random: string; obfuscated: string; bold: string; strikethrough: string; underlined: string; underline: string; italic: string; italics: string; reset: string }
```

Comment:
This types were auto generated (inferred) that's why sometimes they look so weird, will fix